Andrea / exercise-1/stylesheet.css

```diff
@@ -0,0 +1,169 @@
+/* Universal */
+
+*{
+    margin :0;
+    padding : 0;
+    box-sizing : border-box;
+
+}
+
+main{
+    margin-bottom: 32px;
+}
+
+h3 {
+    font-size: 18px;
+    margin: 16px ;
+    text-transform: capitalize;
+}
+
+/* Header*/
+a{
+    color: black;
+    text-decoration: none;
+    text-transform: capitalize;
+}
+
+a img, li img{
+    width: 20px;
+    height: 20px;
+    margin-right: 16px;
+}
+
+li{
+    list-style: none;
+    line-height: 2;
+    text-transform: capitalize;
+}
+
+span {
+    display: block;
+}
+
+/* Especifico */
+.hero {
+    position: relative;
+    width: 100wv;
+    height: 100vh;
+    display: flex;
+    justify-content: center; /* Horizontalmente */
+    align-items: center; /* Verticalmente */
+    
+}
+
+.hero h1 {
+    position: relative;
+    color: white;
+    text-align: center;
+    font-family: "Montserrat", sans-serif;
+    font-size: 55px;
+    font-weight: normal;
+    text-transform: capitalize;
+}
+
+.hero::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: url(hero.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+    filter: brightness(60%);
+}
+
+.section-menu {
+    margin: 0 16px;
```

Comment:
We should also have a margin on top, at least, it looks funky
![image](https://user-images.githubusercontent.com/71283270/101287254-069c4180-37a4-11eb-8d8a-e7165f598230.png)